### PR TITLE
Listing Ciphscoin

### PR DESCRIPTION
**Pull request by ciphs**

### CODE REVIEW

File: qml.qrc [hunk 1]
```diff
@@ -54,6 +54,7 @@
         <file>atomic_defi_design/assets/images/coins/cel.png</file>
         <file>atomic_defi_design/assets/images/coins/cennz.png</file>
         <file>atomic_defi_design/assets/images/coins/chips.png</file>
+		<file>atomic_defi_design/assets/images/coins/ciphs.png</file>
```
Comment (Milerius) [inline qml.qrc:57]: spurious alignment

File: atomic_defi_design/qml/Constants/Style.qml [hunk 2]
```diff
@@ -309,6 +309,7 @@ QtObject {
                                           "ECA": "#A915DC",
                                           "QTUM": "#2E9AD0",
                                           "CHIPS": "#598182",
+                                          "CIPHS": "#ecd900",
```
Comment (Milerius) [inline atomic_defi_design/qml/Constants/Style.qml:312]: Hexadecimal character need to be uppercase

**Review by cipig — APPROVED** [review]

**Review by Milerius — CHANGES_REQUESTED** [review]
see my comment

**Review by ciphs — COMMENTED** [review]
fix

**Pull request merged**